types/npm/index.d.ts

```diff
@@ -167,8 +167,8 @@ declare namespace NPM {
         Conf: ConfigStatic;
         defs: ConfigDefs;
 
-        get(setting: string): string;
-        set(setting: string, value: string): void;
+        get(setting: string): any;
+        set<T>(setting: string, value: T): void;
```

Comment:
This is considered a `useless generic` as it doesn't contribute to type safety at all, please remove it.

Reply:
So should I change to `value: any` ?

Reply:
Yes.

Reply:
Ok done